src/Controls/src/Core/Handlers/Items/Android/MauiRecyclerView.cs

```diff
@@ -440,6 +445,11 @@ protected virtual int DetermineTargetPosition(ScrollToRequestEventArgs args)
 				else if (ItemsViewAdapter.ItemsSource is IGroupableItemsViewSource groupItemSource)
 				{
 					item = FindBoundItemInGroup(args, groupItemSource);
+
+					if (item is null)
+					{
+						return -1;
```

Comment:
Here, can return a named constant instead of literal –1

`const int InvalidPosition = -1;`